codec/address_test.go

```diff
@@ -39,3 +39,79 @@ func TestAddressJSON(t *testing.T) {
 	require.NoError(json.Unmarshal(addrJSONBytes, &parsedAddr))
 	require.Equal(addr, parsedAddr)
 }
+
+func TestEncodeWithChecksum(t *testing.T) {
+	require := require.New(t)
+
+	tests := []struct {
+		address     Address
+		checksumStr string
+	}{
+		{
+			address: CreateAddress(
+				byte(0),
+				ids.Empty,
+			),
+			checksumStr: "0x000000000000000000000000000000000000000000000000000000000000000000a7396ce9",
+		},
+		{
+			address: func() Address {
```

Comment:
Do we need the anonymous functions here or can we just use `CreateAddress` directly?

Reply:
This was a remanent of testing checksumming via EIP-55; removed that test case

Reply:
Removed anonymous functions